frontend/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+import { FormEvent, useState } from "react";
+
+function Header(props: any): JSX.Element {
```

Comment:
I had named it with the idea of it being above the list of tasks. Down the line making child components within it. I changed the name to `Input` or is there a more fitting name?

Reply:
I think that the usual name for components that are on top of other components and store some logic for them is `Wrapper`. Of the two - `Header` and `Input` - Header was actually a better idea. Also, I've then spotted that most of you are using the name `Header`, so don't feel obliged to change it as it will make it harder for you to discuss your code with others.